spec/runtime/api/core-api@tradeshift.com/ts.ui.Notification.spec.js

```diff
@@ -39,7 +39,7 @@ describe('ts.ui.Notification', function likethis() {
 	});
 
 	it('should support markdown for links', function(done) {
-		var text = 'Markdown for (link text)[and_link_url]';
+		var text = 'Markdown for [link text](and_link_url)';
```

Comment:
:warning: But before you merge, perhaps we should make a test for the OLD (invalid) syntax so that we never risk breaking stuff that is already in some database!